tensorboard/plugins/graph/graph_util.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utilities for graph plugin."""
+from tensorboard.compat.proto import function_pb2
+
+
+def combine_graph_defs(to_proto, from_proto):
+  """Combines two GraphDefs by adding nodes from from_proto into to_proto.
+
+  Args:
+    to_proto: a destination GraphDef.
```

Comment:
It may be worth documenting whether these are the TensorFlow protos or
the TensorBoard protos. It looks to me like the code expects TB protos,
but will actually work if the `to_proto` is a TF proto and the
`from_proto` has no nodes and no library, or the `to_proto` is a TB
proto and the `from_proto` is a TF proto with no nodes and an
absent-or-empty library.